client/cli/src/commands/utils.rs

```diff
@@ -202,11 +210,11 @@ where
 	match output {
 		OutputType::Json => {
 			let json = json!({
-				"networkId": String::from(network_override),
+				"networkId": network_override.to_string(),
 				"publicKey": format_public_key::<Pair>(public_key.clone()),
-				"ss58PublicKey": public_key.to_ss58check_with_version(network_override),
+				"ss58PublicKey": public_key.to_ss58check_with_version(network_override).as_str(),
```

Comment:
What's the advantage of `.as_str()`? Is the Display implementation currently inefficient?

Reply:
IIRC, the `json!` macro expects types that can be deferred into some trait/type/struct used by `serde_json`. `ArrayString` isn't supported but `&str` is.